vector/src/test/java/im/vector/app/features/location/LocationDataTest.kt

```diff
@@ -77,7 +80,7 @@ class LocationDataTest {
 
     @Test
     fun unstablePrefixTest() {
-        val geoUri = "geo :12.34,56.78;13.56"
+        val geoUri = "geo :12.34,56.78;u=13.56"
```

Comment:
I think there is a (not new) problem here; the line should be

```suggestion
        val geoUri = "geo:12.34,56.78;u=13.56"
```

This is not a big deal since we are comparing the string. But just for clarity. Or even just use some mockk data like

```kotlin
        val geoUri = "theGeoUri"
```

Reply:
You are right. I change the value of the uri to not use the pattern as it is not mandatory to have it for the purpose of this test. See: [1df71d3](https://github.com/vector-im/element-android/pull/6376/commits/1df71d31e20afff89d844737d140ec6426b98e1c)